### DISCUSSION TITLE
2021-03-12-mkdocs-macros-plugin%E5%AE%8F%E6%8F%92%E4%BB%B6/

### DISCUSSION THREAD

**Discussion by giscus** [Fusyong/Fusyong.github.io#3] [Announcements]
# 2021-03-12-mkdocs-macros-plugin%E5%AE%8F%E6%8F%92%E4%BB%B6/

https://blog.xiiigame.com/2021-03-12-mkdocs-macros-plugin%E5%AE%8F%E6%8F%92%E4%BB%B6/

**Comment by yanedie** [comment]
主要这插件能让mkdocs拥有Python的能力，你可以用Python加载外部文件获取里面的内容注册为变量再引用
